app/src/dbRequest.js

```diff
@@ -0,0 +1,57 @@
+//Task Requests
+export const getTasks = async (user) => {
+  const response = await fetch("/api/tasks", {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      "timedown-user": user,
+    },
+  });
+  return response.json();
```

Comment:
Yeah this is what makes me think you shouldn't need to do `JSON.stringify()` followed immediately by `JSON.parse()` above. 🤔